miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -124,6 +138,7 @@ abstract class MiniAppMessageBridge {
         when (callbackObj.action) {
             ActionType.GET_UNIQUE_ID.action -> onGetUniqueId(callbackObj)
             ActionType.REQUEST_PERMISSION.action -> onRequestPermission(callbackObj)
+            ActionType.REQUEST_DEVICE_PERMISSION.action -> onRequestDevicePermission(callbackObj)
```

Comment:
I don't think we need to change the name of the JavaScript method. In the JS SDK we've encapsulated this call within `requestLocationPermssion`, so the mini app developers never call `requestPermission` directly anyways. I'm afraid that changing the JS SDK name could cause difficulties with backwards compatibility in the future. So this could just be:
```
ActionType.REQUEST_PERMISSION.action -> onRequestDevicePermission(callbackObj)
```
What do you think?

Reply:
Yeah, I have updated this part.